app/assets/stylesheets/components/table/_table-head.scss

```diff
@@ -0,0 +1,66 @@
+//----------------------------------------
+// animations
+//----------------------------------------
+@keyframes showHeader {
+  from {
+    transform: translateY(-100%);
+  }
+  to {
+    transform: translateY(0);
+  }
+}
+
+//----------------------------------------
+// classes
+//----------------------------------------
+.table-head {
+  margin-top: rem-calc(30);
+  height: rem-calc(56);
+
+  display: none;
+
+  @include breakpoint($medium) { display: block; }
+
+  &--stuck {
+    width: rem-calc(1140);
+    
+    position: fixed;
+    top: 0;
+
+    animation: showHeader .25s forwards linear;
+  }
+
+  &__row {
+    @include table-grid-basic;
+  }
+
+  &__cell {
+    font-size: rem-calc(16);
+    font-weight: $bold;
+    padding: rem-calc(16 18);
```

Comment:
I think the padding on table and cells is a bit small, it might need a bit extra on the table x axis